### ISSUE TITLE
/cartesian?

### ISSUE THREAD

**Issue by mzed** [immersive-audio-live/ADM-OSC#19]
Does this message have any meaning? Should we deprecate?

**Comment by lzwicker** [comment]
Imho a receiver of ADM-OSC messages can clearly understand if the sender uses cartesian or polar instructions solely from the commands being used. Therefore this information is redundant.

**Comment by madees** [comment]
Agreed that it seems redundant, and nobody gave us a use case about what to do with this parameter, so it can be deprecated.

**Comment by madees** [comment]
Finally, I found an idea of feature behind such a parameter, to describe production environment context.
So can know if you receive (XY) positions datas produced and listened on an "egocentric sweet spot" (originally polar coordinates) converted to cartesian coordinates, or native cartesians.
Same on the opposite: if you received (a,d) you can differentiate if initial production was monitored as "allocentric" or "physic based" (originally cartesians coordinates) converted to polar, or native polars.
Does that makes any sense to you ?

**Comment by JGaudin** [comment]
Okay, is @madees 's use case worth adding the description ? Feels redundant as well

**Comment by mzed** [comment]
Coordinates in ADM are always specified relative to a sweet spot at 0, 0, 0.  Either polar or cartesian coordinates could be specified, by they always relate to that spot. The possibility of specifying different listener positions is interesting, but I think it's a different feature.
